### ISSUE TITLE
Want testing samples

### ISSUE THREAD

**Issue by KaiWangCanada** [Moicen/RobotWriter#1]
It is a wonderful work. If there were testing samples, like generated texts from training text Afanti, it will be better!

**Comment by Moicen** [comment]
@KaiWangCanada thanks for your comment.

Sorry for late reply, as my vpn didn't work for half a month cause a  congress.
Here is some outputs generated by this program. Apparently they are nonsense, the program still has some bugs, and the algorithm needs to be optimized. I'll continue this work when i get new ideas, and GPUs.
You should clone this repository and run programs by your self. As you'll understand it better.
Any communication about it is welcome. Hope you can have fun with it.
[story.txt](https://github.com/Moicen/RobotWriter/files/1436237/story.txt)

**Comment by KaiWangCanada** [comment]
Thanks for the sample! I tried your code, got similar results. Using more training text and training more episodes should produce better results, but training can take days.

**Comment by Moicen** [comment]
@KaiWangCanada Yes the training consumes too much calculate resource. You'll need a good GPU to train, or you can rent a GPU server if you just want to have fun with it.
